package.json

```diff
@@ -110,7 +110,7 @@
 		"vscode": "^1.1.34",
 		"webpack": "4.35.0",
 		"webpack-cli": "3.3.5",
-		"yarn": "^1.17.0"
+		"yarn": "^1.17.3"
```

Comment:
If we're switching to use `yarn`, it seems weird to specify it as a dependency for itself.

Reply:
Ah yeah, not sure why this was included in the first place.